lib/resources/dstu2/immunization.yaml

```diff
@@ -0,0 +1,79 @@
+---
+name: Immunization
+field_name_base_url: http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization
+fields:
+
+- name: vaccineCode
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: The vaccine that was administered or was to be administered.
+  example: |
+            {"vaccineCode": {
+              "coding": [
```

Comment:
Although... I there there is an argument to be made that I was inconsistent in my own examples. Applying to rules properly would align the { under the c rather than then o

```
{"vaccineCode": {
   "coding": [
    {"system": "http://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=cvx",
     "code": "43"}],
   "text": "Engerix-B (obsolete)"}}
```